### ISSUE TITLE
No mountable files systems MAC

### ISSUE THREAD

**Issue by chexurs** [drawpile/Drawpile#678]
I am able to run older versions of drawpile, but when I download 2.0.7.1 It says there's no mountable file systems. Please HELP!

**Comment by tincancrab** [comment]
Please try version 2.0.10?

**Comment by chexurs** [comment]
I tried 2.0.10-test and 2.0.10-test2 both did not work as well :(

**Comment by FosterHaven** [comment]
I reproduced the issue on my El Capitan system. The dmg is formatted as APFS, which old versions like El Capitan don't support. No recompilation is necessary to fix this issue, simply repackage the build with Mac OS Extended (unless the process is automatic, in which case find out where the code is formatting it as APFS and correct it).

I worked around this issue by copying the app from the dmg using a version of macOS that supports APFS to the El Capitan system using a flash drive, though not everyone has this advantage.

Hope that helps!
